source/pacmap/__init__.py

```diff
@@ -1,5 +1,10 @@
 from .pacmap import PaCMAP, sample_neighbors_pair
 
-import pkg_resources
-__version__ = pkg_resources.get_distribution('pacmap').version
+from importlib.metadata import version, PackageNotFoundError
+
+try:
+    __version__ = version('pacmap')
+except PackageNotFoundError:
+    __version__ = "unknown"
+
 __all__ = ["pacmap"]
```

Comment:
```suggestion
__all__ = ["PaCMAP", "sample_neighbors_pair"]
```

also on second thought, this is what you're exposing.